gips/data/landsat/landsat.py

```diff
@@ -200,6 +220,33 @@ def __init__(self, filename):
                 }
             self.visbands = [col for col in smeta['colors'] if col[0:4] != "LWIR"]
             self.lwbands = [col for col in smeta['colors'] if col[0:4] == "LWIR"]
+        elif c1_match:
+            VerboseOut('C1 asset', 2)
```

Comment:
optional:  `verbose_out` is preferred for new code; we're trying to deprecate things that disobey python naming conventions.